hudi-common/src/main/java/org/apache/hudi/common/table/view/AbstractTableFileSystemView.java

```diff
@@ -269,19 +269,35 @@ public void close() {
 
   /**
    * Clears the partition Map and reset view states.
+   * <p>
+   * NOTE: This method SHOULD NOT BE OVERRIDDEN which may cause stale file system view
+   * to be served.  Instead, override {@link AbstractTableFileSystemView#runReset} to
+   * add custom logic.
    */
   @Override
   public void reset() {
```

Comment:
make this `final`? so it cant be overridden? why add comment :)

Reply:
no longer needed as discussed.  We directly use the write lock in each overriding implementation instead of indirect usage.